presto-native-execution/presto_cpp/main/thrift/ProtocolToThrift.cpp

```diff
@@ -65,10 +65,10 @@ void toThrift(const std::vector<P>& p, std::vector<T>& t) {
 }
 
 void toThrift(const protocol::TaskState& proto, thrift::TaskState& thrift) {
-  thrift = (thrift::TaskState)((int)proto);
+  thrift = (thrift::TaskState)(static_cast<int>(proto));
```

Comment:
Should there be a static_cast from int -> thrift::TaskState enum as well.

Same comment for the line below as well.

Reply:
Hey @aditi-pandit good catch, updated with the change, thank you!